superset-frontend/src/views/CRUD/data/database/DatabaseList.tsx

```diff
@@ -85,6 +101,164 @@ function DatabaseList({ addDangerToast, addSuccessToast }: DatabaseListProps) {
     };
   }
 
+  const initialSort = [{ id: 'changed_on_delta_humanized', desc: true }];
+  const columns = useMemo(
+    () => [
+      {
+        accessor: 'database_name',
+        Header: t('Database'),
+      },
+      {
+        accessor: 'backend',
+        Header: t('Backend'),
+        size: 'xxl',
+        disableSortBy: true, // TODO: api support for sorting by 'backend'
+      },
+      {
+        accessor: 'allow_run_async',
+        Header: (
+          <TooltipWrapper
+            label="allow-run-async-header"
+            tooltip={t('Asynchronous Query Execution')}
+            placement="top"
+          >
+            <span>{t('AQE')}</span>
+          </TooltipWrapper>
+        ),
+        Cell: ({
+          row: {
+            original: { allow_run_async: allowRunAsync },
+          },
+        }: any) =>
+          allowRunAsync ? (
+            <IconBlack name="check" />
+          ) : (
+            <IconBlack name="cancel-x" />
+          ),
+        size: 'md',
+      },
+      {
+        accessor: 'allow_dml',
+        Header: (
+          <TooltipWrapper
+            label="allow-dml-header"
+            tooltip={t('Allow Data Danipulation Language')}
+            placement="top"
+          >
+            <span>{t('DML')}</span>
+          </TooltipWrapper>
+        ),
+        Cell: ({
+          row: {
+            original: { allow_dml: allowDML },
+          },
+        }: any) =>
+          allowDML ? <IconBlack name="check" /> : <IconBlack name="cancel-x" />,
```

Comment:
Nit: this ternary is repeated a number of times with the same display element. Is there an easy way to DRY it up?

Reply:
Good suggestion. Fixed!